platform/mock-local-home-platform.ts

```diff
@@ -38,28 +39,58 @@ export class UDPScanConfig implements ScanConfig {
   }
 }
 
-// TODO(cjdaly): figure out how to direct arbitrary instances of stub classes to an arbitrary
-// instance of MockLocalHomePlatform. May need to use some static event or singleton pattern.
-
 // TODO(cjdaly): add other radio scan support
 export class MockLocalHomePlatform implements MockUDPListener {
+  //  Singleton instance
+  private static instance: MockLocalHomePlatform;
+
   private udpScanConfigs: UDPScanConfig[] = [];
-  private network: MockNetwork;
+  private mockNetwork: MockNetwork;
+  private app: smarthome.App;
+  private deviceManager: smarthome.DeviceManager;
   private localDeviceIds: string[] = [];
 
-  constructor(network: MockNetwork, udpScanConfigs: UDPScanConfig[]) {
-    this.network = network;
+  private constructor() {
+    this.setupUDP();
+  }
+
+  public initializeRadio(
```

Comment:
Will this need to change to accommodate other transport protocols aside from UDP?

Reply:
Yeah, I will need to update this function once I address #18.  I plan to use optional parameters.